### DISCUSSION TITLE
Keyboard shortcut to mark feed as read?

### DISCUSSION THREAD

**Discussion by tomhusband** [FreshRSS/FreshRSS#6572] [Q&A] [answered]
Is there a keyboard shortcut to mark a feed as read and then move to the next feed? Same function as the check mark at the bottom of the feeds. If not is there a way to make a custom shortcut?

I'm really liking FreshRSS and I've tried a bunch of readers.

**Comment by math-GH** [comment, marked as answer]
Have you checked the shortcuts config page?

**Reply by tomhusband** [reply]
Yes, I did but, sorry, I didn't see it. 
edit: Got it. Sorry, I'm a little slow on the uptake.
